_trumps.spacing.scss

```diff
@@ -177,13 +177,13 @@ $inuit-enable-paddings--none:           false !default;
      * Tiny negative margins.
      */
 
-    .#{$inuit-spacing-namespace}u--m--  { margin:           -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mt-- { margin-top:       -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mr-- { margin-right:     -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mb-- { margin-bottom:    -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--ml-- { margin-left:      -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mh-- { margin-right:     -$inuit-margin--tiny !important; margin-left:     -$inuit-margin--tiny !important; }
-    .#{$inuit-spacing-namespace}u--mv-- { margin-top:       -$inuit-margin--tiny !important; margin-bottom:   -$inuit-margin--tiny !important; }
+    .#{$inuit-spacing-namespace}u--m--1\/4  { margin:           -$inuit-margin--tiny !important; }
+    .#{$inuit-spacing-namespace}u--mt--1\/4 { margin-top:       -$inuit-margin--tiny !important; }
```

Comment:
The only places where the base margin is used is the `1x` variant. All others use some kind of `$inuit-margin--variant`. I would argue that it's not necessary to have the `1x` naming at all, but where the other variants are used to use an `x` in the name. Even if we are halving or quartering the value, we're still making the margin in these cases proportional to some other value.